packages/compass/src/main/menu.spec.ts

```diff
@@ -588,6 +590,89 @@ describe('CompassMenu', function () {
       });
     });
 
+    // TODO(COMPASS-8505): Remove this test
+    it('should generate a menu template for linux', async function () {
+      await App.preferences.savePreferences({
+        enableMultipleConnectionSystem: true,
+      });
+      sinon.stub(process, 'platform').value('linux');
+
+      expect(serializable(CompassMenu.getTemplate(0))).to.deep.equal([
+        {
+          label: '&Connections',
+          submenu: [
+            { label: '&Import Saved Connections' },
+            { label: '&Export Saved Connections' },
+            { type: 'separator' },
+            { label: 'E&xit' },
+          ],
+        },
+        {
+          label: 'Edit',
+          submenu: [
+            { label: 'Undo', role: 'undo' },
+            { label: 'Redo', role: 'redo' },
+            { type: 'separator' },
+            { label: 'Cut', role: 'cut' },
+            { label: 'Copy', role: 'copy' },
+            { label: 'Paste', role: 'paste' },
+            {
+              label: 'Select All',
+              role: 'selectAll',
+            },
+            { type: 'separator' },
+            { label: 'Find' },
+            { type: 'separator' },
+            { label: '&Settings' },
+          ],
+        },
+        {
+          label: '&View',
+          submenu: [
+            { label: '&Reload' },
+            { label: '&Reload Data' },
+            { type: 'separator' },
+            { label: 'Actual Size' },
+            { label: 'Zoom In' },
+            { label: 'Zoom Out' },
+          ],
+        },
+        {
+          label: '&Help',
+          submenu: [
+            { label: `&Online ${app.getName()} Help` },
+            { label: '&License' },
+            { label: `&View Source Code on GitHub` },
+            { label: `&Suggest a Feature` },
+            { label: `&Report a Bug` },
+            { label: '&Open Log File' },
+            { type: 'separator' },
+            { label: `&About ${app.getName()}` },
+            { label: 'Check for updates…' },
+          ],
+        },
+      ]);
+    });
+
+    it('does not crash when rendering menu item with an accelerator', async () => {
+      const menu = Menu.buildFromTemplate([
+        {
+          label: 'Test Super',
+          accelerator: 'Super+Ctrl+T',
+        },
+        {
+          label: 'Test Meta',
+          accelerator: 'Meta+Ctrl+T',
+        },
+      ]);
```

Comment:
Can we use our template here so that we're checking that _our_ template works on all platforms? 🙂 (this will probably fail now because this includes accelerator and will run on linux)

Reply:
i added more checks and also skipped the tests for linux

Reply:
I think there is some kind of misunderstanding here: the fix you're making is programmatically changing the template generated by our code in a way that should make it usable on linux, I want us to have a test that checks that the generated template actually works on linux (not tangentially checking it through something arbitrary like whether or not acceletator key is present), more specifically: I want this test to build our template and then run it through the logic in this test that would trigger the break if we're not doing it correctly and it should run on linux too as we're expecting the introduced change to make it work on linux. Does this make sense?

Reply:
Yeah i see you point and there was misunderstanding from my side. I am testing the actual menu now [383ff25](https://github.com/mongodb-js/compass/pull/6485/commits/383ff25ba249f8b2213a1285feee89c4bda44f60). I am not setting the menu on the app, directly trying to show it which if crashes would fail the test .